pkg/cmd/openshift/openshift.go

```diff
@@ -81,6 +81,8 @@ func CommandFor(basename string) *cobra.Command {
 		cmd = kubernetes.NewCommand(basename, basename, out)
 	case "origin", "atomic-enterprise":
 		cmd = NewCommandOpenShift(basename)
+	case "pod-diagnostics":
```

Comment:
So I see `openshift infra` doesn't show up to users unless they know to look for it (nor was I aware of it). `openshift infra pod-diagnostics`? I can nuke the symlink.